extensions/wikia/UserProfilePageV3/css/UserProfilePage_modal.scss

```diff
@@ -15,13 +16,22 @@
 
 	.input-group {
 		float: none;
-		position: relative;
 
 		input[type="text"],
 		.favorite-wikis {
 			@include box-sizing-border-box;
 			width: 100%;
 		}
+
+		.tweet-at, .fb-start {
+			float: left;
+			margin-top: 10px;
+			width: 110px;
+		}
+
+		input[name="fbPage"], input[name="twitter"] {
```

Comment:
I think this css needs to be tweaked a little bit. When I go to enter my facebook profile link the cursor starts to the left of the trailing /.
![screen shot 2015-03-02 at 11 18 37 am](https://cloud.githubusercontent.com/assets/1657792/6448359/3685a880-c0ce-11e4-8c14-8b21041d3e3f.png)